gensim/test/test_ldamodel.py

```diff
@@ -242,7 +243,7 @@ def testGetDocumentTopics(self):
             self.assertTrue(isinstance(topic, list))
             for k, v in topic:
                 self.assertTrue(isinstance(k, int))
-                self.assertTrue(isinstance(v, float))
+                self.assertTrue(np.issubdtype(v, float))
```

Comment:
simple `isinstance` is better here (and everywhere).

Reply:
simple `isinstance` fails cause `float32` is not an instance of 'float'